tests/profiler/test_profiler.py

```diff
@@ -617,3 +620,14 @@ def test_profile_callbacks(tmpdir):
         e.name == "[pl][profile][Callback]EarlyStopping{'monitor': 'train_loss', 'mode': 'min'}.on_validation_start"
         for e in pytorch_profiler.function_events
     )
+
+
+@pytest.mark.skipif(
+    not (_TORCH_GREATER_EQUAL_1_11 and hasattr(torch.profiler, "_KinetoProfile")),
```

Comment:
Isn't `torch.profiler._KinetoProfile` always defined in torch>=1.11?

Reply:
yes. you are correct